english/advice_from_mother/advice_from_mother.md

```diff
@@ -3,4 +3,9 @@ such as yourself should avoid LSD. Knowing now that this is all a bad
 trip, you use the bonfire to melt all the marshmallows, ignoring the 
 giant pile of fluff in one corner, watching the swirl and glitter of 
 the toasting surface until you can walk out of the corner and back 
-into the bar.
\ No newline at end of file
+into the bar.
+
+As you walk back into the bar, you wonder whether you ought to
+
+[do yoga](../yoga/yoga.md)
+[make cookies](../bake-cookies/cookies.md)
```

Comment:
There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.